components/input-validation-mgt/org.wso2.carbon.identity.input.validation.mgt/src/main/java/org/wso2/carbon/identity/input/validation/mgt/model/handlers/UsernameValidationConfigurationHandler.java

```diff
@@ -156,4 +160,50 @@ private String getUsernameRegEx(RealmConfiguration realmConfig) {
         return new String();
     }
 
+    /**
+     * Perform post actions after validation configuration are updated.
+     *
+     * @param tenantDomain      Tenant Domain.
+     * @param configuration     Updated validation configuration.
+     */
+    @Override
+    public void postValidationConfigurationUpdateHandler(String tenantDomain, ValidationConfiguration configuration)
```

Comment:
Rename the method.. check previous comment

Reply:
Addressed

Reply:
Not addressed

Reply:
Addressed